cMake/FreeCAD_Helpers/PrintFinalReport.cmake

```diff
@@ -127,24 +117,29 @@ macro(PrintFinalReport)
         endif(DEFINED PySide_FOUND)
     endif(DEFINED MACPORTS_PREFIX)
 
-    if(DEFINED Shiboken2_FOUND)
-        message(STATUS "Shiboken2:           ${Shiboken2_VERSION} [${Shiboken2_DIR}] [${SHIBOKEN_INCLUDE_DIR}]")
-    else(DEFINED Shiboken2_FOUND)
-        message(STATUS "Shiboken2:           not found")
-    endif(DEFINED Shiboken2_FOUND)
-    if(DEFINED PySide2_FOUND)
-        message(STATUS "PySide2:             ${PySide2_VERSION} [${PYSIDE_INCLUDE_DIR}]")
-        if(NOT PYSIDE_INCLUDE_DIR)
-            message(STATUS " IncludeDir:         Unable to find, python version mismatch?")
-        endif(NOT PYSIDE_INCLUDE_DIR)
-    else(DEFINED PySide2_FOUND)
-        message(STATUS "PySide2:             not found")
-    endif(DEFINED PySide2_FOUND)
-    if(DEFINED PYSIDE2_TOOLS_FOUND)
-        message(STATUS "PySide2Tools:        [${PYSIDE2UICBINARY}] [${PYSIDE2RCCBINARY}]")
-    else(DEFINED PYSIDE2_TOOLS_FOUND)
-        message(STATUS "PySide2Tools:        not found")
-    endif(DEFINED PYSIDE2_TOOLS_FOUND)
+    if (FREECAD_QT_MAJOR_VERSION EQUAL 5)
+        set(PYSIDE_VERSION_MAJOR 2)
```

Comment:
No, but it will be 😄 -- it's part of my plans for the new FinalReport code, but I started writing that and remembered I hadn't actually committed the previous changeset yet. I thought I undid back far enough in time, but I missed a line. Sorry for the confusion.